src/main/resources/application.yaml

```diff
@@ -0,0 +1,12 @@
+spring:
+  jpa:
+    database: postgresql
+    show-sql: true
+    hibernate:
+      ddl-auto: create
+  datasource:
+    url: jdbc:postgresql://localhost:8081/userapi
+    username: karen
```

Comment:
It would better to use a more agnostic username, for example, `userapi_app`
Consider it as a name for application as a database user

Reply:
Sorry, I didn't remove the username, it was just for the test.